common/src/main/java/com/microsoft/identity/common/internal/controllers/BaseController.java

```diff
@@ -123,6 +124,13 @@ public abstract AcquireTokenResult acquireTokenSilent(final AcquireTokenSilentOp
                     acquireTokenOperationParameters.getRequestHeaders()
             );
 
+            // We don't want to show the SELECT_ACCOUNT page if login_hint is set.
+            if (acquireTokenOperationParameters.getLoginHint() != null &&
```

Comment:
We have a handy convenience function for checking non-`null`ness as well as length (see `StringExtensions.isNullOrBlank()`)

Reply:
or we can use Android's in build lib, `TextUtils.isEmpty()`

Reply:
@kreedula +1 we should transition to this (low pri)